src/main/java/com/github/krzysiekjodlowski/ox/Event.java

```diff
@@ -0,0 +1,16 @@
+package com.github.krzysiekjodlowski.ox;
+
+/**
+ * Describes generic event, and it's associated meta data which get
+ * sent in the bus to be dispatched to interested Subscribers.
+ *
+ * @author Krzysztof Jodlowski
+ */
+public interface Event<T> {
+  /**
+   * Returns the stored data associated with the event.
+   *
+   * @return the stored data
+   */
+  T getData();
```

Comment:
No usage of this method. What is the point of this interface?

Reply:
Removed, interface described as marker interface.